### ISSUE TITLE
Lack of captions for videos

### ISSUE THREAD

**Issue by tsvoronos** [rstudio-education/primers#109]
All videos in the RStudio primers hosted on Vimeo lack any closed captioning. I’ve had to stop using these primers for my course since students with accessibility needs are unable to follow the content of the videos. This is a significant oversight with respect to accessibility of this content. I would be happy to help create and add captions if necessary.
